### PR TITLE
Update multi-node cluster configuration documentation

### PR DESCRIPTION
**Pull request by vaikuntht**
Updated documentation for configuring a multi-node Qserv cluster. Also made a few typo changes in other documentation.

### CODE REVIEW

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 1]
```diff
@@ -1,38 +1,72 @@
...
 
-.. code-block:: bash
+2. Configure Qserv on master and all workers:
+
+- Run :code:`qserv-configure.py -i -R <qserv-run directory>`
+
+- Edit :code:`<qserv-run directory>/qserv-meta.conf`
+ 
+  - Change :code:`node_type` to :code:`master` or :code:`worker`
+  - | Change master DNS name to the name of the machine as shown by :code:`hostname -s`.
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: change to: "Change master DNS name to the name of the master machine as shown by..."

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 2]
```diff
@@ -1,38 +1,72 @@
...
+    | Do not alter it in any way, for example, by adding the full domain name.
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: change to ".. for example, do not add the full ..."

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 3]
```diff
@@ -1,38 +1,72 @@
...
+    | On the worker, use the master's DNS name, **not** the worker's
+
+- Rerun :code:`qserv-configure` without :code:`-i` as :code:`qserv-configure.py -R <qserv-run directory>`
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: change to " without :code:`-i`:"

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 4]
```diff
@@ -1,38 +1,72 @@
...
+Pre-requisites
+==============
 
-Build/install Qserv on each machine of the cluster, using eups.
+Build/install Qserv and its pre-requisites on all nodes.
 
-Install master and worker nodes
-===============================
+Configure master and worker nodes
+=================================
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: add a note somewhere at the beginning that the examples here are for 2-worker cluster.

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 5]
```diff
@@ -1,38 +1,72 @@
...
+    | On the worker, use the master's DNS name, **not** the worker's
+
+- Rerun :code:`qserv-configure` without :code:`-i` as :code:`qserv-configure.py -R <qserv-run directory>`
+
+3. Copy secret file from the master to every worker:
+
+   *Note: secret file is read when qserv-wmgr client starts*
+
+   :code:`scp <qserv-run directory>/etc/wmgr.secret> <worker1 hostName>:<qserv-run directory>/etc/wmgr.secret`
+   :code:`scp <qserv-run directory>/etc/wmgr.secret> <worker2 hostName>:<qserv-run directory>/etc/wmgr.secret`
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: remove extra ">" after secret

File: doc/source/HOW-TO/cluster-configuration.rst [hunk 6]
```diff
@@ -1,38 +1,72 @@
...
+     CREATE NODE worker1 type=worker port=5012 host=<worker1 hostName>;
+     CREATE NODE worker2 type=worker port=5012 host=<worker2 hostName>;
+
+Run Integration Test
+====================
+
+.. code-block:: bash
 
+  qserv-check-integration.py --case=01 --load -M --mode=qserv
+  qserv-check-integration.py --case=02 --load -M --mode=qserv
```
Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: add "..." or spell out or 5

File: doc/source/HOW-TO/cluster-configuration.rst (hunk 5 above)
Comment (vaikuntht) [inline doc/source/HOW-TO/cluster-configuration.rst]: Oops, nice catch.

File: doc/source/HOW-TO/cluster-configuration.rst (hunk 6 above)
Comment (vaikuntht) [inline doc/source/HOW-TO/cluster-configuration.rst]: I think "..." will be better in case we have more than 5 test cases in the future.

Comment (jbecla) [inline doc/source/HOW-TO/cluster-configuration.rst]: ok

**Pull request merged**